### DISCUSSION TITLE
Вопрос по вебтунам/манхвам

### DISCUSSION THREAD

**Discussion by east-mist** [lirix360/ReadmangaGrabber#33] [Ideas]
Столкнулся с проблемой при скачивании глав с ридманги.
Дело в том, что в некоторых манхвах завезли режим вебтуна, т.е. все изображения отображаются бесшовно на одной странице.
Примеры: 
1) https://readmanga.io/grad_molnii__A5327/vol1/1
2) https://readmanga.io/grad_molnii__A5327/vol1/1
Поведение программы: то же, что и при "бане" со стороны рида.
Есть какое-то решение, которое я пропустил?

**Comment by lirix360** [comment]
Скачивание подобного формата на данный момент не поддерживается.
